18F/C2
# [Delivers #108105962] Add "purchase type" to 18F procurements

* Required field
* Using ActiveRecord enum

* To test:
  log in as a user with 18f client slug and create a new request
  "purchase type" field should be required
  "purchase type" should show up on Proposals#show and in email body

* Story: https://www.pivotaltracker.com/story/show/108105962
